pkg/env/local_notifiers.go

```diff
@@ -0,0 +1,6 @@
+package env
+
+var (
+	// SecuredClusterNotifiers defines if certain notifiers are configured to send messages through the secured cluster.
```

Comment:
SecuredClusterNotifiers controls whether notifications to certain notifiers are sent through the secured cluster or not.

Reply:
tweaked the message, rebased on master, renamed the file to be more in line with naming scheme